webpack/ForemanInventoryUpload/Components/PageHeader/components/SettingsWarning/SettingsWarning.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Alert, AlertActionCloseButton } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const SettingsWarning = ({
+  autoUpload,
+  hostObfuscation,
+  isCloudConnector,
+}) => {
+  const [showUploadWarning, setShowUploadWarning] = useState(true);
+  const [showObfuscationWarning, setShowObfuscationWarning] = useState(true);
+  if (
+    !isCloudConnector ||
+    (!setShowUploadWarning && !setShowObfuscationWarning)
```

Comment:
just wondering, is there a chance the sets won't exist in this line?

Reply:
It was suppose to be `showObfuscationWarning` and not `setShowObfuscationWarning`, fixed, thanks!